config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
```

Comment:
The example you provide is caused by the heredoc notation (i.e. the first argument) being on the same line as the opening paren (see http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ClosingParenthesisIndentation).  What they are trying to enforce is that the closing paren should be lined up with the open paren when the arguments are as well.

With this configuration, the following kind of code would be considered acceptable
```ruby
other_method(arg1,
             arg2
                             )
```

```ruby
    other_method(arg1,
                 arg2
)
```

Disabling this rule means that we are allowing that kind of argument/parenthesis alignment, and I'm not sure I agree with that.

If you think about the value of that heredoc just being an argument (`arg1` in the example method above) that happens to take multiple lines, I think the example you've provided should be written along the lines of, which seems preferable to me.

```ruby
expect(result.to_xml).to be_equivalent_to(
  <<-EOF
    <fileGroupDifference>
      <subset change="identical" count="3">
      ...
  EOF
)
```